pkg/build/gobuild.go

```diff
@@ -255,12 +255,17 @@ func build(ctx context.Context, ip string, dir string, platform v1.Platform, con
 		return "", fmt.Errorf("could not create env for %s: %w", ip, err)
 	}
 
-	tmpDir, err := ioutil.TempDir("", "ko")
+	tmpDir, err := ioutil.TempDir(os.TempDir(), "ko")
```

Comment:
I'm not sure this change is necessary, at least not for the goal of this PR.

Reply:
okay, I reverted that change.